yarn.lock

```diff
@@ -0,0 +1,3789 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
@DanStory When I was downloading the package using npm the second time the npm install failed because the package-lock.json had some errors so I preferred to use yarn to avoid this problem.

Reply:
I would suggest we fix the npm package-lock errors vs introducing a different package management that could require  additional build pipeline and developer environment changes.

Simplest resolution would be to delete the `package-lock.json` and run `npm install` to regen it.